### PR TITLE
Increment table MIB entries - Reviewer EM

### PR DESCRIPTION
**Pull request by johadalin**
Additions to the MIB for the AllTimersTable, counting global counts of all timers in each chronos node. 
These have been put under the OID Chronos 4.

### CODE REVIEW

File: PROJECT-CLEARWATER-MIB [hunk 1]
```diff
@@ -5050,6 +5050,80 @@ chronosInvalidTimersProcessedCount OBJECT-TYPE
     DESCRIPTION "The total number of invalid timers processed over the period."
     ::= { chronosInvalidTimersProcessedEntry 2 }
 
+chronosAllTimersTable OBJECT-TYPE
```
Comment (eleanor-merry) [inline PROJECT-CLEARWATER-MIB]: You need to also add these to the ChronosGroup - see https://github.com/Metaswitch/clearwater-snmp-handlers/blob/master/PROJECT-CLEARWATER-MIB#L5161

Comment (eleanor-merry) [inline PROJECT-CLEARWATER-MIB]: @rkd-msw changed the MIB file to fix up various warnings in https://github.com/Metaswitch/clearwater-snmp-handlers/pull/93. Can you make the equivalent changes to this, and run smilint over the result.

Comment (rkd-msw) [inline PROJECT-CLEARWATER-MIB]: Note that the Jenkins build will now fail if it doesn't pass smilint.

File: PROJECT-CLEARWATER-MIB [hunk 2]
```diff
@@ -5014,6 +5014,81 @@ chronosInvalidTimersProcessedCount OBJECT-TYPE
...
+                 timers on this Chronos node, measured
+                 and weighted over time scopes."
+    ::= { chronos 4 }
+
+chronosAllTimersEntry OBJECT-TYPE
+    SYNTAX      ChronosAllTimersEntry
+    MAX-ACCESS  not-accessible
+    STATUS      current
+    DESCRIPTION "The average, variance, LWM, HWM and
+                 count of all timers on this Chronos
```
Comment (eleanor-merry) [inline PROJECT-CLEARWATER-MIB]: Should be actions not count

**Comment by eleanor-merry** [comment]
One comment, otherwise looks good

**Comment by eleanor-merry** [comment]
Looks good

**Pull request merged**